src/Microsoft.AspNetCore.Mvc.Razor.Extensions/Microsoft.AspNetCore.Mvc.Razor.Extensions.csproj

```diff
@@ -11,6 +11,16 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\Microsoft.AspNetCore.Razor.Language\BoundAttributeDescriptorComparer.cs">
+      <Link>BoundAttributeDescriptorComparer.cs</Link>
+    </Compile>
+  </ItemGroup>
```

Comment:
This isn't great either, do these types implement `IEquatable<>`? 

Think about it this way. Someone besides you and me doesn't have the ability to just include this file. What should that person do?